server/src/kandidatsøk/kandidatsøk.ts

```diff
@@ -48,113 +47,3 @@ export const harTilgangTilKandidatsøk: RequestHandler = async (request, respons
             );
     }
 };
-
-export const leggTilAuthorizationForKandidatsøkEs =
```

Comment:
Bare for å være på den heeeelt sikre siden, så er tilsvarende auditlogging gjort i backend nå?

Reply:
Ja :)